WMF Framework/Event Platform/EventPlatformClient.swift

```diff
@@ -751,3 +726,12 @@ public protocol EventInterface: Codable {
      */
     static var schema: EventPlatformClient.Schema { get }
 }
+
+// MARK: Objective-C
+
+public protocol MetricsClientObjectiveCCompat: BackgroundFetcher, PeriodicWorker {
```

Comment:
It doesn't look like this protocol is used anywhere, can you remove it? Your extensions in `MetricsClientBridge.swift` should be enough to conform to `BackgroundFetcher` and `PeriodicWorker`.

Reply:
Done!